media/js/firefox/new.js

```diff
@@ -235,13 +237,15 @@
                 gaTrack(['_trackPageview', virtual_url]);
             }
 
+            var params = new _SearchParams();
```

Comment:
Any reason to call this multiple times in the one file? Could it just be called once at the beginning?